### PR TITLE
docs: update install and get started link in readme.md

### LINKED ISSUE #7831: docs: installation-guide.md and getting-started.md are hard to read
### Current State

 In the documentation's [get-started](https://github.com/apache/apisix#get-started ) part, links of [install documentation](https://github.com/apache/apisix/blob/master/docs/en/latest/installation-guide.md) and [Getting Started](https://github.com/apache/apisix/blob/master/docs/en/latest/getting-started.md) are hard to read(guess due to the JSX part in MD file is not properly rendered)

### Desired State

Change link to APISIX Documentation site. 
 https://apisix.apache.org/docs/apisix/installation-guide/ and https://apisix.apache.org/docs/apisix/getting-started/

### PR DESCRIPTION
**Pull request by monkeyDluffy6017**
### Description

In the documentation's [get-started](https://github.com/apache/apisix#get-started) part, links of [install documentation](https://github.com/apache/apisix/blob/master/docs/en/latest/installation-guide.md) and [Getting Started](https://github.com/apache/apisix/blob/master/docs/en/latest/getting-started.md) are hard to read(guess due to the JSX part in MD file is not properly rendered)

Fixes #7831

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/README.md [hunk 1]
```diff
@@ -167,11 +167,11 @@ A/B 测试、金丝雀发布（灰度发布）、蓝绿部署、限流限速、
 
 1. 安装
 
-   请参考[APISIX 安装指南](./installation-guide.md)。
+   请参考[APISIX 安装指南](https://apisix.apache.org/docs/apisix/installation-guide/)。
 
 2. 入门指南
 
-   入门指南是学习 APISIX 基础知识的好方法。按照 [入门指南](getting-started.md)的步骤即可。
+   入门指南是学习 APISIX 基础知识的好方法。按照 [入门指南](https://apisix.apache.org/docs/apisix/getting-started/)的步骤即可。
```
Comment (juzhiyuan) [inline docs/zh/latest/README.md:174]:
```suggestion
   入门指南是学习 APISIX 基础知识的好方法。按照 [入门指南](https://apisix.apache.org/zh/docs/apisix/getting-started/)的步骤即可。
```

File: docs/zh/latest/README.md [hunk 2]
```diff
@@ -167,11 +167,11 @@ A/B 测试、金丝雀发布（灰度发布）、蓝绿部署、限流限速、
 
 1. 安装
 
-   请参考[APISIX 安装指南](./installation-guide.md)。
+   请参考[APISIX 安装指南](https://apisix.apache.org/docs/apisix/installation-guide/)。
```
Comment (juzhiyuan) [inline docs/zh/latest/README.md:170]:
```suggestion
   请参考[APISIX 安装指南](https://apisix.apache.org/zh/docs/apisix/installation-guide/)。
```

File: docs/zh/latest/README.md (hunk 1 above)
Comment (monkeyDluffy6017) [reply docs/zh/latest/README.md]: Done

File: docs/zh/latest/README.md (hunk 2 above)
Comment (monkeyDluffy6017) [reply docs/zh/latest/README.md]: Done

**Review by SylviaBABY — APPROVED** [review]

**Review by hf400159 — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**